scanner/e2etests/helmchart/values.yaml

```diff
@@ -0,0 +1,12 @@
+# Default values for helm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  tag: latest
```

Comment:
We currently don't have a `latest` pushed anywhere that I know of?

Reply:
I just needed a default. You are not supposed to interact with the chart directly, but through the `make` targets, so doesn't matter.